apps/explorer/lib/explorer/chain/token.ex

```diff
@@ -87,10 +87,23 @@ defmodule Explorer.Chain.Token do
     end
   end
 
-  def join_with_transfers(queryable \\ Token) do
+  def join_with_transfers(address_hash, queryable \\ Token) do
+    # Use a CTE so that postgres filters the token_transfers table by the to_address_hash and from_address_hash first.
+    # Otherwise, it will also filter it by token_contract_address_hash, and this makes the query prohibitively slow.
     from(
       t in queryable,
-      join: tt in TokenTransfer,
+      join:
+        tt in fragment(
+          """
```

Comment:
Credo is asking for a sigil here because of the amount of double quotes.

```suggestion
          ~s"""
```

Reply:
They are not really necessary, it's just a more "canonical" way of writing the query. The double quotes are needed when your column is case sensitive or in similar situations. When you print the query (for example using `Ecto.Adapters.SQL.to_sql`), it will show like this for that reason. I'm OK with removing them if it makes it more readable.

Reply:
Removing those quotes also resolve your credo problem without adding `~s`, so I would 👍 to removing them.